api/http/api_inventory.go

```diff
@@ -593,8 +599,16 @@ func (i *inventoryHandlers) PatchDeviceAttributesInternalHandler(
 		u.RestErrWithLog(w, r, l, err, http.StatusBadRequest)
 		return
 	}
-	for _, a := range attrs {
+	for i, a := range attrs {
 		a.Scope = r.PathParam("scope")
```

Comment:
@kjaskiewiczz this will never overwrite the element in the attrs array. I know it is not the part of this TR, but i just noticed that.

Reply:
since you noticed, maybe you can fix it? ;)

Reply:
https://github.com/mendersoftware/inventory/pull/456 only for you Baby.